.idea/.idea.VTTiny/.idea/.gitignore

```diff
@@ -0,0 +1,13 @@
+﻿# Default ignored files
```

Comment:
One minor nitpick, can we not merge this into the outermost .gitignore file? Do we need this folder to exist?

Reply:
It does need to exist and be committed for Rider, for some reason rider does not like normal gitignores